DeveloperDocs.md

```diff
@@ -7,25 +7,95 @@
 ```
 pip install .[dev]
 ```
-* Copy the file `.secrets/dtenv_template.sh` to `.secrets/dtenv.sh` if you will be using a cloud-based database, or `.secrets/dtenv_localdb.sh` if you will run a local postgres server using docker, and populate this file with values for the various environment variables (ask an existing developer for these).   Note that if using the docker option, you will need to have Docker installed and running on your machine.
 
-### Running the backend API
+## Running an Instance of DTBase
+
+DTBase has three main components:
+
+- Postgresql Database
+- Backend API
+- Frontend API
+
+All these components can be run either locally or deployed via Azure. The following sections contain instructions for running an instance of dtbase either locally or via Azure.
+
+### Local Deployment
+
+#### Running a Local Database
+
+The postgresql database can be run via a locally deployed container with Docker.
+
+1. Copy the file `.secrets/dtenv_template.sh` to `.secrets/dtenv_localdb.sh`
+2. Populate this file with following variables:
+
+```
+#!/bin/bash
+
+# Test database
+export DT_SQL_TESTUSER="postgres"
+export DT_SQL_TESTPASS="password"
+export DT_SQL_TESTHOST="localhost"
+export DT_SQL_TESTPORT="5432"
+export DT_SQL_TESTDBNAME="dt_dev"
```

Comment:
I think we should have different names for the two databases, otherwise running the test suite will overwrite what you have in your dev db.

Reply:
Good spot. Changed to "test_db"